components/match2/wikis/rainbow_six/brkts_wiki_specific.lua

```diff
@@ -297,6 +297,7 @@ function matchFunctions.getDateStuff(match)
 	else
 		match.date = lang:formatDate('c', _EPOCH_TIME)
 		match.dateexact = false
+		match.nodate = true
```

Comment:
```suggestion
		match.hasNoDate = true
```

Reply:
In addition to camelcase, also removed the negation in the name (match.hasDate) and adjusted the setting of it based on the new name. 
Reason for removing the negation is that I don't like double negatives, takes longer for the brain to parse (`not match.hasNoDate` vs `match.hasDate`)

Reply:
Cool, completely agree with your amendment of my suggestion :)